src/components/Banner/Banner.mdx

```diff
@@ -234,6 +235,12 @@ renders once.
 
 <Canvas of={BannerStories.Dismissible} />
 
+## With HTML content passed as string
+
+HTML content passed as a string is rendered as HTML, useful in cases where HTML content is stored in an environment variable.
```

Comment:
Can you add, "... or from other sources where the content is stored as strings."